test/extended/testdata/test-auth-build.yaml

```diff
@@ -28,3 +32,7 @@ objects:
           name: ruby:latest
           namespace: openshift
       type: Source
+    output:
+      to:
+        kind: ImageStreamTag
+        name: output:latest
```

Comment:
i take it the lack of a push secret is why we didn't see this issue before.....  normally we try to avoid pushing during tests since it wastes time but i guess we can live with one.  can you add a comment about why we're pushing here so we don't purge this output in a future test optimization though?